Makefile

```diff
@@ -159,15 +159,15 @@ docs/generate/%:
 	mv -f $(README_TMP) $*
 	@ echo "[$@]: Documentation files creation complete!"
 
-docs/lint/%:
+docs/lint/%: | terraform/lint
```

Comment:
Did you mean to put this on the /% target?

Reply:
negatory, updated